dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -267,8 +268,12 @@ def runValidation(self, job_id, interfaces):
 
         rowNumber = 1
         fileType = jobTracker.getFileType(job_id)
+        # Get orm model for this file
+        model = [ft.model for ft in FILE_TYPE if ft.name == fileType][0]
```

Comment:
I'm not sure what I think of this syntax, but I've seen some folks use it before:

```python
model = next(ft.model for ft in FILE_TYPE if ft.name == fileType)
```